### PR TITLE
Check DG

### PR DESCRIPTION
**Pull request by yleeyilin**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 <img src="images/ArchitectureDiagram.png" width="280" />
 
-The ***Architecture Diagram*** given above explains the high-level design of the App.
+The ***Architecture Diagram*** above explains the high-level design of the PoochPlanner.
```
Comment (jannaleong) [inline docs/DeveloperGuide.md:33]: of PoochPlanner. no need the

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -42,10 +42,10 @@ Given below is a quick overview of the main components and how they interact wit
 
 The bulk of the app's work is done by the following four components:
 
-* [**`UI`**](#ui-component): The UI of the App.
+* [**`UI`**](#ui-component): The UI of the PoochPlanner.
```
Comment (jannaleong) [inline docs/DeveloperGuide.md:45]: same here dont need the

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -42,10 +42,10 @@ Given below is a quick overview of the main components and how they interact wit
...
 * [**`Logic`**](#logic-component): The command executor.
-* [**`Model`**](#model-component): Holds the data of the App in memory.
-* [**`Storage`**](#storage-component): Reads data from, and writes data to, the hard disk.
+* [**`Model`**](#model-component): Holds the data of the PoochPlanner in memory.
```
Comment (jannaleong) [inline docs/DeveloperGuide.md:47]: same here dont need the

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -58,9 +58,9 @@ The *Sequence Diagram* below shows how the components interact with each other f
...
-* implements its functionality using a concrete `{Component Name}Manager` class (which follows the corresponding API `interface` mentioned in the previous point.
+* implements its functionality using a concrete `{Component Name} Manager` class which follows the corresponding API `interface` mentioned in the previous point.
 
-For example, the `Logic` component defines its API in the `Logic.java` interface and implements its functionality using the `LogicManager.java` class which follows the `Logic` interface. Other components interact with a given component through its interface rather than the concrete class (reason: to prevent outside component's being coupled to the implementation of a component), as illustrated in the (partial) class diagram below.
+For example, the `Logic` component defines its API in the `Logic.java` interface and implements its functionality using the `LogicManager.java` class which follows the `Logic` interface. Other components interact with a given component through its interface rather than a concrete class. This is to prevent outside component's being coupled to the implementation of a component, as illustrated in the (partial) class diagram below.
```
Comment (jannaleong) [inline docs/DeveloperGuide.md:63]: prevent outside component's from being coupled...

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -72,15 +72,15 @@ The **API** of this component is specified in [`Ui.java`](https://github.com/AY2
 
 ![Structure of the UI Component](images/UiClassDiagram.png)
 
-The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class which captures the commonalities between classes that represent parts of the visible GUI.
+The UI consists of a `MainWindow` that are made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class, which captures the commonalities between classes that represent parts of the visible GUI.
```
Comment (jannaleong) [inline docs/DeveloperGuide.md:75]: is

**Review by jannaleong — APPROVED** [review]
lgtm but minor errors

**Review by chiageng — APPROVED** [review]
LGTM

**Pull request merged**